man/systemd.netdev.xml

```diff
@@ -179,6 +179,9 @@
           <row><entry><varname>fou</varname></entry>
           <entry>Foo-over-UDP tunneling.</entry></row>
 
+          <row><entry><varname>xfrm</varname></entry>
+          <entry>A virtual tunnel interface like vti/vti6 but with several advantages.</entry></row>
```

Comment:
btw a link to some document explaining this in more detail (via `<ulink>`) is often very useful